### PR TITLE
chore: update ja.json

### PR DESCRIPTION
**Pull request by dai**
Merge branch 'RSSNext:dev' into dev

### PR Type

- [ ] Feature
- [ ] Bugfix
- [ ] Hotfix
- [ ] Other (please describe):

### Changelog

- [ ] I have updated the changelog/next.md with my changes.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -80,5 +80,11 @@
...
   "i18n-ally.localesPaths": ["locales"],
   "i18n-ally.namespace": true,
-  "i18n-ally.pathMatcher": "{namespaces}/{locale}.json"
+  "i18n-ally.pathMatcher": "{namespaces}/{locale}.json",
+  "[json]": {
+    "editor.defaultFormatter": "vscode.json-language-features"
+  },
+  "[jsonc]": {
+    "editor.defaultFormatter": "vscode.json-language-features"
+  }
```
Comment (hyoban) [inline .vscode/settings.json:89]: Why modify this? We are using prettier to format json

Comment (dai) [reply .vscode/settings.json]: I changed it on my PC browser, is that ok?
Anyway, thanks again for all your reviews. I'm learning a lot.

Comment (hyoban) [reply .vscode/settings.json]: We have ci to check the formatting of the code, you can modify the settings locally.

**Comment by hyoban** [comment]
Thank you very much for your continued contribution to Follow's i18n

**Pull request merged**